README.md

```diff
@@ -1705,6 +1708,8 @@ The following extractors use this feature:
 
 NOTE: These options may be changed/removed in the future without concern for backward compatibility
 
+*1: **Warning**: Using this feature may result in your account termination. Use it at your own risk.
```

Comment:
just move it alongside the option description

#### niconico
* `segment_duration`: Segment duration in milliseconds for DMC (HLS) formats. Use it at your own risk since this feature **may result in your account termination**